src/resdk/resolwe.py

```diff
@@ -485,6 +493,7 @@ class ResAuth(requests.auth.AuthBase):
     def __init__(self, username=None, password=None, url=DEFAULT_URL):
         """Authenticate user on Resolwe server."""
         self.logger = logging.getLogger(__name__)
+        self.cookies = {}
```

Comment:
Why define empty dict here if you re-create it in the lines below (line 524)?

Reply:
To always have cookies defined. If no username and password is given, code will return None before it reaches the last line in `__init__`.